docs/sandbox-permissions.rst

```diff
@@ -56,6 +56,8 @@ for portals can refer to the `xdg-desktop-portal API documentation
 <https://flatpak.github.io/xdg-desktop-portal/>`_ for
 information on how to use them.
 
+.. _sandbox-permissions-guidelines:
+
```

Comment:
this is not needed use ref like this https://github.com/flatpak/flatpak-docs/blob/4589636295d0f3de5a8d8b6f98ee750e8938e4d0/docs/debugging.rst?plain=1#L61

Reply:
Nice, thanks! I'm not very familiar with Sphinx/RST, so I appreciate your help! :blush: 

I've updated it accordingly.